### PR TITLE
New package: batsignal-1.0.0

### PR DESCRIPTION
**Pull request by juantascon**

### CODE REVIEW

**Comment by ndowens** [comment]
Add CFLAGS="-I${XBPS_CROSS_BASE}/usr/include" and this may fix the cross-compile

File: srcpkgs/batsignal/template [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+license="ISC"
+homepage="https://github.com/electrickite/batsignal"
+distfiles="https://github.com/electrickite/batsignal/archive/${version}.tar.gz"
+checksum=4087fe29c0ecebe69d000ba5cc26e038b28872f39a50d38e815b5d46c6b0f81d
+
+post_install() {
+	vlicense LICENSE
+}
+
+pre_configure() {
```
Comment (travankor) [inline srcpkgs/batsignal/template]: pre_configure goes before post_install

File: srcpkgs/batsignal/template [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Template file for 'batsignal'
+pkgname=batsignal
+version=1.0.0
+revision=1
+archs="noarch"
```
Comment (travankor) [inline srcpkgs/batsignal/template]: get rid of this

**Comment by ericonr** [comment]
Merged in d2ac638a7a654942d20

**Pull request closed without merging**